### ISSUE TITLE
Accessibility-relevant HTML-tags in JSX

### ISSUE THREAD

**Issue by sabinammm** [liqd/adhocracy4#918]
**URL:** https://meinberlin-dev.liqd.net/ideas/2021-00628/ See DATES in comments from a4
**user:** any
**expected behaviour:** accessible date through html tag
**behaviour:** 
**important screensize:**
**device & browser:** 
**Comment/Question:** 
In reference to https://github.com/liqd/a4-meinberlin/issues/4017, is this also generally the case in the projects for html rendered by jsx-files that we are missing the accessibility-relevant html-tags @phillimorland, i.e. can you estimate the scope of this issue?  In the above case, it should be changed to include `<time datetime=".xxxx"></time> ` right?
Screenshot?

![image](https://user-images.githubusercontent.com/17978375/145200173-c1ecf6bb-4a8d-46fd-92f6-890f7223d4b9.png)

**Comment by phillimorland** [comment]
after discussion concerning this and checking of various sources, it was decided that a machine readable date was not required for comments, it is possible that time tag could be used without the datetime attribute, this is to be considered source https://html.spec.whatwg.org/#the-time-element this issue is now considered closed
